aiohttp/http_writer.py

```diff
@@ -130,6 +130,7 @@ class PayloadWriter(AbstractPayloadWriter):
     def __init__(self, stream, loop, acquire=True):
         self._stream = stream
         self._transport = None
+        self._buffer = []
```

Comment:
I tried to benchmark bytearray vs join in #2179, but the results were inconclusive at best (or lean in favor of `b''.join`). I'll try to dig up why asyncio changed again to bytearray